app/services/bibdata.rb

```diff
@@ -53,9 +53,18 @@ def holding_locations
 
     private
 
+      def parse_holding_locations(response:)
```

Comment:
These were introduced due to errors which I encountered within my development environment (and on CircleCI).